README.md

```diff
@@ -25,13 +25,14 @@ A new version numbering scheme is also being introduced, during the development
 
 The table below shows the exact version of ACS deployed with each chart version/tag.
 
-| Chart Version/Tag | Default | 6.2.N | 6.1.N | Community |
-|--|--|--|--|--|
-| 5.0.0-M1 | 7.0.0-M2 | 6.2.2 | 6.1.1 | 6.2.1-A8 |
-| 5.0.0-M2 | 7.0.0-M3 | 6.2.2 | 6.1.1 | 7.0.0-A10 |
-| 5.0.0 | 7.0.0 | 6.2.2 | 6.1.1 | 7.0.0 |
-| 5.0.1 | 7.0.0 | 6.2.2 | 6.1.1 | 7.0.0 |
-| 5.1.0-M1 | 7.1.0-M1 | 6.2.2 | 6.1.1 | 7.1.0-M1 |
+| Chart Version/Tag | Default | 7.0.N | 6.2.N | 6.1.N | Community |
+|--|--|--|--|--|--|
+| 5.0.0-M1 | 7.0.0-M2 | | 6.2.2 | 6.1.1 | 6.2.1-A8 |
+| 5.0.0-M2 | 7.0.0-M3 | | 6.2.2 | 6.1.1 | 7.0.0-A10 |
+| 5.0.0 | 7.0.0 | | 6.2.2 | 6.1.1 | 7.0.0 |
+| 5.0.1 | 7.0.0 | | 6.2.2 | 6.1.1 | 7.0.0 |
+| 5.1.0-M1 | 7.1.0-M1 | | 6.2.2 | 6.1.1 | 7.1.0-M1 |
+| 5.1.0-M2 | 7.1.0-M2 | 7.0.1 | 6.2.2 | 6.1.1 | 7.1.0-M2 |
 
```

Comment:
I fixed 2 layout issues in this table.

Question: Why is the Community version not 7.0.1 in the last two rows when we are about to release Community and Enterprise 7.0.1. I assume we are releasing 7.0.1 Community, but we need to check.